apps/web-giddh/src/app/new-inventory/component/new-inventory-advance-search/new-inventory-advance-search.component.html

```diff
@@ -148,7 +148,7 @@ <h4 class="modal-title pull-left">{{ localeData?.advance_search_filter?.advance_
                             >
                                 {{ commonLocaleData?.app_search }}
                             </button>
-                            <button id="clear-btn" class="btn btn-danger mr-0" (click)="advanceSearchAction('clear')">
+                            <button id="clear-btn" mat-stroked-button class="mr-0" (click)="advanceSearchAction('clear')">
```

Comment:
add color

Reply:
nisha, we don't add color on clear/cancel buttons